examples/workflows/failing_activities_workflow.rb

```diff
@@ -2,7 +2,21 @@
 
 class FailingActivitiesWorkflow < Temporal::Workflow
   def execute(count)
-    futures = (0..count - 1).map { |i| GuessActivity.execute(i) }
+    fail_callbacks = {}
```

Comment:
nit: `Hash.new { |_k, _v| Set.new }` will set a default value for missing keys, so you don't need to call `|| Set.new` later